F1WM/Repositories/NewsRepository.cs

```diff
@@ -99,5 +160,27 @@ public async Task<bool> IncrementViews(int id)
 			await context.SaveChangesAsync();
 			return true;
 		}
+
+		private NewsSummaryPaged GetPagedResult(IEnumerable<News> dbNews, int page, int countPerPage)
```

Comment:
Please change the first param type to IQueryable and evaluate the query only when you need to take the elements (method Take), and count all news (method Count on all news)